app/lib/features/chat_ng/widgets/chat_editor/chat_editor.dart

```diff
@@ -151,6 +163,18 @@ class _ChatEditorState extends ConsumerState<ChatEditor> {
       final fallbackPlain = draft.plainText();
       if (fallbackPlain.trim().isNotEmpty) {
         textEditorState.replaceContent(fallbackPlain, htmlBody);
+      } else {
+        // FOR DESKTOP/SIDE VIEW:
+        // it means the draft is empty, but still valid document (e.g. empty paragraph) for editor to get auto focused
+        // set selection to null to avoid the editor from being focused
```

Comment:
so from Appflowy perspective, an empty editor still contains valid empty node, which editor would load initially with selection (auto-focus) by entering room. We simply check here with having desktop (sidebar) view to set selection to null if draft is empty.